rpc/README.md

```diff
@@ -4343,6 +4343,8 @@ RPC Module Pool for transaction memory pool.
 
 Submits a new transaction into the transaction pool. If the transaction is already in the pool, rebroadcast it to peers.
 
+It should be noted that `send_transaction` is asynchronous. That is, the return of the transaction hash, does NOT mean that the transaction is fully verified. If you need to follow up on the status of the transaction, it is recommended to use the `get_transaction` rpc.
```

Comment:
Please note `send_transaction` is asynchronous.

Reply:
The return value of the request does NOT mean the transaction is fully verified. If you want to follow up on the status of the transaction, please use the `get_transaction` rpc to query the latest status.

Reply:
I ask ChatGPT to reword this. It give me:
```
    /// Please note that `send_transaction` is an asynchronous process.
    /// The return of `send_transaction does NOT indicate that the transaction have been fully verified.
    /// If you want to track the status of the transaction, please use the `get_transaction`rpc.

```

Reply:
Simple and short words is a better style.